### PR TITLE
Pass the strlen to the readCallback as was expected

### PR DESCRIPTION
**Pull request by rullzer**
So turns out I was to quick with https://github.com/icewind1991/Streams/pull/8

Fixed now
And adjusted the tests to actually check for this

### CODE REVIEW

**Comment by rullzer** [comment]
I have no idea why php <7 is failing

**Comment by rullzer** [comment]
Ok turns out php <7 always reads the chunk size and does other magic in fread.

File: src/CallbackWrapper.php [hunk 1]
```diff
@@ -95,9 +95,10 @@ public function stream_open($path, $mode, $options, &$opened_path) {
 	}
 
 	public function stream_read($count) {
+		var_dump($count);
```
Comment (icewind1991) [inline src/CallbackWrapper.php]: :see_no_evil:

**Comment by icewind1991** [comment]
you can add a condition for the php version in the test case.

As long as the callback actually gets the number of bytes read it doesn't matter that those values depend on php version

File: src/CallbackWrapper.php (hunk 1 above)
Comment (rullzer) [reply src/CallbackWrapper.php]: Fixed

**Pull request merged**